src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -27,4 +27,19 @@ public partial class AppDomain
             }  
         }  
     }
-}
\ No newline at end of file
+}
+namespace System.Runtime.ExceptionServices
+{
+    public class FirstChanceExceptionEventArgs : EventArgs
+    {
+        private Exception m_Exception;
+        public FirstChanceExceptionEventArgs(Exception exception)
+        {
+             m_Exception = exception;
+        }
+        public Exception Exception
+        {
+            get { return m_Exception; }
+        }
```

Comment:
Nit: If you want to take advantage of C# 6, this could be:
```C#
public class FirstChanceExceptionEventArgs : EventArgs
{
    public FirstChanceExceptionEventArgs(Exception exception)
    {
        this.Exception = exception;
    }

    public Exception Exception { get; }
}
```